src/utils/worklets/identifiers_highlighting.cljs

```diff
@@ -0,0 +1,31 @@
+(ns utils.worklets.identifiers-highlighting)
```

Comment:
Same quesiton as in the JS file: why we don't create those JS functions here in ClojureScript?

Reply:
This was the way it was done when I joined the project and I've just been following this practice. Perhaps @briansztamfater or @Parveshdhull can provide a reason.

Reply:
https://github.com/status-im/status-mobile/issues/13452

Reply:
Thank you so much @Parveshdhull for giving context. I think the current solution is good for now.

Now Reanimated 3 has been released but I think we are using 2.11, so do you think this will be improved when we migrate to 3?

Reply:
Reanimated 3 also uses worklets, so we still have to rely on javascript workaround. Or we can look into cljs alternatives for creating worklets (not sure about performance impact though)
https://github.com/roman01la/cljs-worklet

Reply:
Ok, it seems the JS approach is good enough. Thanks @Parveshdhull